### PR TITLE
Fix F3DOperators

### PR DESCRIPTION
**Pull request by Meakk**

### CODE REVIEW

File: src/library/F3DOperators.h [hunk 1]
```diff
@@ -31,7 +31,14 @@ static std::locale comma_locale(std::locale::classic(), new comma_ctype);
 template<typename T>
 std::ostream& operator<<(std::ostream& stream, const std::vector<T>& vector)
 {
-  std::copy(vector.begin(), vector.end(), std::ostream_iterator<T>(stream, ","));
+  if (vector.size() > 0)
+  {
+    stream << vector[0];
+    for (auto it = std::next(vector.begin()); it != vector.end(); it++)
```
Comment (mwestphal) [inline src/library/F3DOperators.h:37]: unless Im missing smething, you are adding vector[0] two times ?

File: src/library/F3DOptions.cxx [hunk 2]
```diff
@@ -197,7 +181,7 @@ F3DOptions ConfigurationOptions::GetOptionsFromArgs(std::vector<std::string>& in
     this->DeclareOption(grp1, "edges", "e", "Show cell edges", options.Edges);
     this->DeclareOption(grp1, "trackball", "k", "Enable trackball interaction", options.Trackball);
     this->DeclareOption(grp1, "progress", "", "Show progress bar", options.Progress);
-    this->DeclareOption(grp1, "up", "", "Up direction", options.Up, true, "[-X|+X|-Y|+Y|-Z|+Z]");
+    this->DeclareOption(grp1, "up", "", "Up direction", options.Up, true, true, "[-X|+X|-Y|+Y|-Z|+Z]");
```
Comment (mwestphal) [inline src/library/F3DOptions.cxx:184]: how could this even work before ?

File: src/library/F3DOptions.cxx [hunk 3]
```diff
@@ -58,27 +59,10 @@ class ConfigurationOptions
   }
 
   template<class T>
-  static std::string ToString(T currValue)
+  static std::string ToString(const T& currValue)
```
Comment (mwestphal) [inline src/library/F3DOptions.cxx:62]: I'm not against it, but this class will be reworked from the ground up, or even removed in this form

File: src/library/F3DOperators.h (hunk 1 above)
Comment (Meakk) [reply src/library/F3DOperators.h:37]: `std::next(vector.begin())`

File: src/library/F3DOptions.cxx (hunk 2 above)
Comment (Meakk) [reply src/library/F3DOptions.cxx:184]: It wasn't working

Comment (Meakk) [reply src/library/F3DOptions.cxx:184]: `arg` was displayed instead of `[-X|+X|-Y|+Y|-Z|+Z]`

File: src/library/F3DOptions.cxx (hunk 3 above)
Comment (Meakk) [reply src/library/F3DOptions.cxx:62]: That's fine if we remove it later. It's just a clean-up.

File: src/library/F3DOperators.h (hunk 1 above)
Comment (mwestphal) [reply src/library/F3DOperators.h:37]: right !

**Review by mwestphal — APPROVED** [review]

**Comment by Meakk** [comment]
@mwestphal I reverted the changes for `ToString`. For some reason it was introducing two failing tests with ASan.

**Pull request merged**